peewee_async.py

```diff
@@ -298,15 +298,6 @@ def execute_sql(self, *args, **kwargs):
                         (str(args), str(kwargs)))
         return super().execute_sql(*args, **kwargs)
 
-    async def fetch_results(self, query, cursor):
```

Comment:
Could you write a couple of tests for CompoundSelectQuery and RawQuery?

Reply:
@kalombos Tests added!